src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+@import "./stars";
+
+.card {
+  width: 200px;
+  padding: 32px 16px 15px;
+  border: 1px solid $color-light;
+  border-radius: $main-radius;
+
+  &__img {
+    width: 159px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__photo {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  &__title {
+    @include normalText;
+    font-weight: 500;
+    color: $color-main-text;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    
+    color: $color-add-text;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    margin: 0 0 24px;
+
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__stars {
+    margin: 0;
+  }
+
+  &__reviews {
+    margin: 0;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 12px;
+    color: $color-main-text;
+  }
+
+  &__pricing {
+    margin: 0 0 16px;
+
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__pricing-title {
+    @include normalText;
+    margin: 0;
+    font-weight: 400;
+    color: $color-add-text;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-main-text;
+  }
+
+  &__btn {
+    display: inline-block;
+    width: 166px;
+    padding: 10px 10px 11px;
+    font-weight: 700;
+    font-size: 14px;
+    color: #fff;
+    background-color: $color-accent;
+    border: 1px solid $color-accent;
+    border-radius: $main-radius;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;

```

Comment:
add transition for bg-color,color and border color

Reply:
fix @Dikey945 comment

Reply:
you didn't add transition to the main properties